xbmc/addons/AddonDatabase.h

```diff
@@ -129,7 +129,9 @@ class CAddonDatabase : public CDatabase
   /*! Clear internal fields that shouldn't be kept around indefinitely */
   void OnPostUnInstall(const std::string& addonId);
 
-  void SyncInstalled(const std::set<std::string>& ids, const std::set<std::string>& enabled);
+  void SyncInstalled(const std::set<std::string>& ids,
+                     const std::set<std::string>& system,
+                     const std::set<std::string>& optional);
```

Comment:
Please try avoid vertical alignment in the future as it breaks formatting and vcs on every change. it's in our guidelines as well now.